cartesi-rollups/api/backend/introduction.md

```diff
@@ -11,9 +12,9 @@ This is accomplished by calling a set of HTTP endpoints, as illustrated by the f
 
 First, the backend retrieves a new request as follows:
 
-- Finish — Communicates that any previous processing has been completed and that the backend is ready to handle the next request. This next request is returned as the call's response and can be of the following types:
+- Finish — Communicates that any previous processing has been completed and that the backend is ready to handle the subsequent request. This following request is returned as the call's response and can be of the following types:
 
-  - **Advance** — Provides input to be processed by the backend to advance the application's state. When processing an `Advance` request, the backend can call the methods `/voucher`, `/notice`, and `/report`. For such requests, the input data contains the payload and metadata, such as the account address that submitted the input.
+  - **Advance** — Provides input to be processed by the backend to advance the application's state. When processing an `Advance` request, the backend can call the methods `/voucher,` `/notice,` and `/report.` For such requests, the input data contains the payload and metadata, such as the account address that submitted the input.
```

Comment:
the "application's state" here should be "the cartesi machine state", but I still do think that people will understand better if you say it as it is right now.

Reply:
Why is this resolved?